### PR TITLE
[WC-2565] Add run e2e automation README file

### PR DESCRIPTION
**Pull request by leonardomendix**
### Pull request type

Test related change (New E2E test, test automation, etc.)

---

### Description

Add a README file explaining more details about the structure and how to execute the e2e test.

### CODE REVIEW

**Review by gjulivan — APPROVED** [review]

File: automation/run-e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+4. Commit the changes in the mpr file;
+5. Execute the `e2e` or `e2edev` script to use the updated project.
+
+## Updating the visual screenshots baseline
+
+In Github Action, we run the runtime and mxbuild in Docker containers. This means that the visual screenshot test is based on a Linux machine, resulting in differing pixel resolutions when running the `e2e` script on a Windows or Mac machine.
+
+Currently, we don't have a script that updates these screenshots independently of your machine, so we have 2 options to address this:
+
+1. Commit your changes to GitHub and wait for the screenshot to fail. This will generate an artifact with the failed screenshot, which you can download. Then, place these image files in the e2e snapshots folders, following the same file name pattern <snapshotName-chromium-linux.png>. Commit these files to have a green build.
```
Comment (gjulivan) [inline automation/run-e2e/README.md:57]: can you be more specific on where this artifact can be found? links, perhaps?

File: automation/run-e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+2. Alternatively, you can use a Linux Virtual Machine locally, using Parallels or Virtual Box. For this, you can use Ubuntu 22.04 and follow the steps described in the "Running E2E locally" guide. The only difference is using the script `e2e` instead of `e2edev`.
+
+Please note that as part of the necessary environment, you need to install `pnpm`, `node`, and `docker`.
```
Comment (gjulivan) [inline automation/run-e2e/README.md:61]: this should be on the top, on pre-requisite sections

File: automation/run-e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Please make sure to follow these steps:
+
+1. Run `pnpm install` before anything else, and add `GITHUB_TOKEN` to your environment variables to download the test project from the other repository.
+2. Run `pnpm run e2edev` and follow the instructions in the terminal.
+3. Locate the desired widget path in the test project.
+4. Run the test project in Studio Pro.
+5. Once the project is running, proceed with the next step of the script in the terminal to verify if the desired test project port is listening. If it is, the Playwright GUI will be initiated.
+6. In the Playwright GUI interface, select the test spec and execute. From there, you can debug or fix the test case in your IDE.
+
```
Comment (gjulivan) [inline automation/run-e2e/README.md:29]: is there a way to run 1 single test independently?

File: automation/run-e2e/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+│   └── testProject
+├── package.json
+└── playwright.config.cjs
+```
+
+## Executing E2E tests locally
+
+Please make sure to follow these steps:
+
+1. Run `pnpm install` before anything else, and add `GITHUB_TOKEN` to your environment variables to download the test project from the other repository.
```
Comment (gjulivan) [inline automation/run-e2e/README.md:23]: add links to https://github.com/settings/tokens

File: automation/run-e2e/README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+├── tests
+│   └── testProject
+├── package.json
+└── playwright.config.cjs
+```
+
+## Executing E2E tests locally
+
+Please make sure to follow these steps:
+
```
Comment (gjulivan) [inline automation/run-e2e/README.md:22]: I think the first instruction should be to "cd" to desidered widgetPackage folder, right?

File: automation/run-e2e/README.md (hunk 1 above)
Comment (leonardomendix) [reply automation/run-e2e/README.md]: ✅

File: automation/run-e2e/README.md (hunk 2 above)
Comment (leonardomendix) [reply automation/run-e2e/README.md]: ✅

File: automation/run-e2e/README.md (hunk 4 above)
Comment (leonardomendix) [reply automation/run-e2e/README.md]: Thanks! 

✅  Added link suggested.

File: automation/run-e2e/README.md (hunk 5 above)
Comment (leonardomendix) [reply automation/run-e2e/README.md:51]: ✅  Fixed

**Review by gjulivan — APPROVED** [review]

**Review by samuelreichert — APPROVED** [review]

**Review by gjulivan — APPROVED** [review]

**Pull request merged**